core/src/main/scala/kafka/security/authorizer/AclEntry.scala

```diff
@@ -100,9 +100,10 @@ object AclEntry {
     resourceType match {
       case ResourceType.TOPIC => Set(READ, WRITE, CREATE, DESCRIBE, DELETE, ALTER, DESCRIBE_CONFIGS, ALTER_CONFIGS)
       case ResourceType.GROUP => Set(READ, DESCRIBE, DELETE)
-      case ResourceType.CLUSTER => Set(CREATE, CLUSTER_ACTION, DESCRIBE_CONFIGS, ALTER_CONFIGS, IDEMPOTENT_WRITE, ALTER, DESCRIBE)
+      case ResourceType.CLUSTER => Set(CREATE, CLUSTER_ACTION, DESCRIBE_CONFIGS, ALTER_CONFIGS, IDEMPOTENT_WRITE, ALTER, DESCRIBE, CREATE_TOKENS, DESCRIBE_TOKENS)
```

Comment:
Do we need this? We have not defined these operations in KIP https://cwiki.apache.org/confluence/display/KAFKA/KIP-373%3A+Allow+users+to+create+delegation+tokens+for+other+users#KIP373:Allowuserstocreatedelegationtokensforotherusers-Operations

Reply:
Oh, thanks for pointing it out, I think it wasn't intentional. Will remove it.